dialogue-target/src/main/java/com/palantir/dialogue/Response.java

```diff
@@ -26,6 +28,9 @@
     /** The HTTP response code for this response. */
     int code();
 
+    /** The HTTP headers for this response. */
+    Map<String, List<String>> headers();
```

Comment:
Does it make sense to provide both this and `Optional<String> contentType();`?

Reply:
I replaced it with `Optional<String> getFirstHeader(String header)` which seems like the nice behavior you'd want